app/views/devise/registrations/_personal_details.html.erb

```diff
@@ -41,21 +44,23 @@
   <% end %>
 
   <% @identifier_schemes.each do |scheme| %>
-    <div class="form-group col-xs-8">
-      <% if scheme.name == 'shibboleth' %>
-          <label class='control-label'>
-            <span class="aria-only" aria-hidden="false"><%= scheme.description.capitalize %></span>
-            <i class="fa fa-user" title="<%= scheme.description.capitalize %>" aria-hidden="true"></i>
-          </label>
-      <% else %>
-          <%= label_tag(:scheme_name, scheme.name.capitalize, class: 'control-label') %>
-      <% end %>
-      <div class='identifier-scheme'>
-        <%= render partial: "external_identifier_#{scheme.name}",
-                   locals: { scheme: scheme,
-                             id: current_user.identifier_for(scheme)} %>
+    <% if scheme.name == 'shibboleth' && current_user.org.present? %>
+      <div class="form-group col-xs-8">
+        <% if scheme.name == 'shibboleth' %>
+            <label class='control-label'>
+              <span class="aria-only" aria-hidden="false"><%= _('Institutional credentials') %></span>
+              <i class="fa fa-user" title="<%= _('Institutional credentials') %>" aria-hidden="true"></i>
+            </label>
+        <% else %>
```

Comment:
The above check to see if the `scheme.name` is shibboleth makes this unreachable.  
Should this be the else to the outer if on line 47?

Reply:
yes. great catch.